packages/techdocs-cli/src/lib/mkdocsServer.test.ts

```diff
@@ -32,7 +32,7 @@ describe('runMkdocsServer', () => {
     it('should run docker directly by default', async () => {
       await runMkdocsServer({});
 
-      const quotedCwd = `"${process.cwd()}":/content`;
+      const quotedCwd = `${process.cwd()}:/content`;
```

Comment:
Maybe we can delete this line and move `${process.cwd()}:/content` down and replace quotedCwd below, as its not a quoted cwd anymore

Reply:
Good catch, improved 🙌🏻